### PR TITLE
Add JIT warmup of lz4.

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth PR

### CODE REVIEW

**Review by bratseth — APPROVED** [review]
Fine for testing it but I think we should move it into compressor so that we can say compressor.warmup(time).

File: configdefinitions/src/vespa/dispatch.def [hunk 1]
```diff
@@ -48,6 +48,9 @@ numJrtTransportThreads int default=8
 # Number of JRT connections per backend node
 numJrtConnectionsPerNode int default=8
 
+# Number of seconds to spend warming up code to prevent JIT cold start issues.
+warmuptime double default=0.1
```
Comment (bratseth) [inline configdefinitions/src/vespa/dispatch.def]: Any particular reason for the double level of defaults? Just set 5 here?

Comment (baldersheim) [reply configdefinitions/src/vespa/dispatch.def]: Intention to avoid long unit test runtime.

**Comment by baldersheim** [comment]
Moved into compressor.

**Pull request merged**